util/pom.xml

```diff
@@ -29,6 +29,52 @@
   <artifactId>util</artifactId>
   <version>1.9.8-SNAPSHOT</version>
 
+  <properties>
+    <hadoop.one.version>0.23.11</hadoop.one.version>
```

Comment:
Do you still use this Hadoop version, or this is just to make it compile?

Reply:
I looked at https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-common to check what corresponds to V1 for hadoop-common. It looks that the timeline matched with other v1 for e.g. hadoop-core, but if you know any other version, I would be happy to change to that. (BTW, we don't use hadoop1)